app/dashboard/views.py

```diff
@@ -680,7 +680,16 @@ def sync_web3(request):
                 print('* getting bounty')
                 bounty = get_bounty(bounty_id, network)
                 print('* processing bounty')
-                did_change, _, _ = web3_process_bounty(bounty)
+                did_change = False
+                max_tries_attempted = False
+                counter = 0
+                while not did_change and not max_tries_attempted:
+                    did_change, _, _ = web3_process_bounty(bounty)
+                    if not did_change:
+                        print("RETRYING")
```

Comment:
Can we drop the print or did you want to keep it for logging purposes?

Reply:
i think its helpful for logging.. almost all of the sync_web3 logging should probably be moved to logging.debug at some point in the near future